hmiServerDev.sh

```diff
@@ -12,42 +42,50 @@ case ${1} in
 	start)
 		COMMAND="start"
 		;;
+	start-server-intellij)
```

Comment:
Thinking more on this, I think we should avoid using `intellij` here, because this could be used with any ide or also just run seperately to up the containers?

Reply:
changed to `start-server-ide`.